### PR TITLE
fix: route module changes and documents pagination

### PR DESCRIPTION
**Pull request by FH-sandeepkaur1**

### CODE REVIEW

File: src/app/pages/dispatch/new-documents/my-documents/my-document-list.component.ts [hunk 1]
```diff
@@ -262,4 +290,84 @@ export class MyDocumentListComponent implements OnInit {
...
   }
+
+  initDataTable() {
+    let current = this;
+    
+    this.serviceUrl = "documents/fetch-records?value1=";
+
+    // if (filters === 'yes') {
+    //   this.filterValues.category = 'tripNo';
+    //   this.serviceUrl = this.serviceUrl + '&searchValue=' + this.filterValues.searchValue + "&startDate=" + this.filterValues.startDate + "&endDate=" + this.filterValues.endDate + "&category=" + this.filterValues.category + "&value1=";
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/new-documents/my-documents/my-document-list.component.ts]: If commented code is not needed please remove them

File: src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts [hunk 2]
```diff
@@ -208,14 +156,67 @@ export class AddRouteComponent implements OnInit {
...
+      let result = await this.hereMap.geoCode(item.stopName);
+      console.log('result', result);
+      this.newCoords.push(`${result.items[0].position.lat},${result.items[0].position.lng}`)
+    }));
+    this.hereMap.calculateRoute(this.newCoords);
+    this.spinner.hide();
+    this.newCoords = [];
   }
 
+  // fetchCountries() {
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts]: Please remove commented code if not needed

File: src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts [hunk 3]
```diff
@@ -300,16 +289,20 @@ export class AddRouteComponent implements OnInit {
     this.errors = {};
     this.hasError = false;
     this.hasSuccess = false;
-
+    console.log("this.routeData", this.routeData);
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts:267]: Remove console.log

File: src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts [hunk 4]
```diff
@@ -335,36 +327,120 @@ export class AddRouteComponent implements OnInit {
...
+    this.routeData.stops.splice(length - 1, 0, item);
+    if (length > 2) {
+      console.log('length if', length);
+      this.routeData.stops[this.routeData.stops.length - 1].stopName = this.routeData.destinationInformation['destinationAddress'];
+    } else {
+      console.log('length else', length);
+      this.routeData.stops[length].stopName = this.routeData.destinationInformation['destinationAddress'];
+    }
+
+    console.log('this.routeData.stops', this.routeData.stops);
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/add-route/add-route.component.ts:405]: please remove console.log

File: src/app/pages/dispatch/permanent-routing/edit-route/edit-route.component.ts [hunk 5]
```diff
@@ -440,10 +433,173 @@ export class EditRouteComponent implements OnInit {
...
 
   mapShow() {
-    // this.mapView = true;
     setTimeout(() => {
       this.hereMap.mapInit();
     }, 100);
   }
 
+  async assignLocation(elem, label) {
+    console.log("dff", elem, label);
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/edit-route/edit-route.component.ts:407]: remove console.log

File: src/app/pages/dispatch/permanent-routing/edit-route/edit-route.component.ts [hunk 6]
```diff
@@ -440,10 +433,173 @@ export class EditRouteComponent implements OnInit {
...
+    const result = await this.hereMap.geoCode(label);
+    const labelResult = result.items[0];
+    console.log("labelResult", labelResult);
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/permanent-routing/edit-route/edit-route.component.ts:410]: remove console.log

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Hi @FH-sandeepkaur1 Please address the review comments.

**Comment by kunalfleethawks** [comment]
Hi @FH-sandeepkaur1 Can you update the PR title with the meaningful message as discussed yesterday

**Review by kunalfleethawks — APPROVED** [review]
Looks good.

**Review by amritfleethawks — APPROVED** [review]

**Pull request merged**